src/modules/CreateDrops/Fields/ArtworkInput.tsx

```diff
@@ -0,0 +1,70 @@
+import { Controller, useFormContext } from 'react-hook-form';
+import { useState, useEffect } from 'react';
+
+import { ImageFileInput } from '@/common/components/ImageFileInput';
+
+import { CreateNftDropFormFieldTypes } from '../NftDrop/CreateNftDropForm';
+import { CreateTicketFieldsSchema } from '../TicketDrop/CreateTicketDropContext/CreateTicketDropContext';
+
+type CreateDropFieldTypes = CreateNftDropFormFieldTypes | CreateTicketFieldsSchema;
+interface ArtworkInputProps {
+  name?: 'artwork' | 'additionalGift.poapNft.artwork';
```

Comment:
do suggest if there's a better idea, looks hacky to me

Reply:
Do you want to try this:
```